app/models/Secret.py

```diff
@@ -0,0 +1,29 @@
+from uuid import uuid4
+
+from .Metadata import Metadata
+
+
+class Secret():
+    def __init__(self, secret: str):
+        self.id = uuid4()
+        self.value = secret
```

Comment:
can we avoid to use the id name since in Python we've a builtin function? can we avoid all method and provide a simple pydantic model with public fields?